### ISSUE TITLE
Missing field in simplifySparqlResults?

### ISSUE THREAD

**Issue by braco** [maxlath/wikidata-sdk#33]
Just curious, why is personLabel removed in this simplifySparqlResults output?

```
  SELECT DISTINCT ?personLabel ?occupation ?twitterName ?instagramUsername
  WHERE {
    ?person wdt:P106 ?occupation.
    OPTIONAL {
      ?person wdt:P2002 ?twitterName.
      ?person wdt:P2003 ?instagramUsername.
    }
    ?occupation wdt:P279* wd:Q5.
    SERVICE wikibase:label { bd:serviceParam wikibase:language "en". }
  }
```

Results are 
```
1 [ { occupation: 'Q947873',¬
  2     twitterName: 'shannamoakler',¬
  3     instagramUsername: 'shannamoakler' },¬
  4   { occupation: 'Q970153',¬
  5     twitterName: 'yoshida_riko',¬
  6     instagramUsername: 'ai_yoshikawa_official' },¬
  7   { occupation: 'Q970153',¬
  8     twitterName: 'Maisie_Williams',¬
  9     instagramUsername: 'maisie_williams' },¬
 10   { occupation: 'Q947873',¬
 11     twitterName: 'VanessaLachey',¬
 12     instagramUsername: 'vanessalachey' },¬
 13   { occupation: 'Q17125263',¬
 14     twitterName: 'coollike',¬
 ...
```

**Comment by maxlath** [comment]
because the `Label`, `Description`, and `AltLabel` suffixed variables try to be joined to there associated variable: try also selecting `?person` and you will get an object with the person qid as `value` and a `label`. But I guess that in absence of this associated variable, it should just behave as other variables

**Comment by maxlath** [comment]
@braco the patch 22ce21c was published in `v5.11.2`, can you confirmed it is working for you?
